docs/graphql/core/migrations/advanced/resetting-migrations.rst

```diff
@@ -0,0 +1,80 @@
+.. meta::
+   :description: Resetting Hasura migrations
+   :keywords: hasura, docs, migration, reset migrations, clear migrations
+
+.. _reset_migration:
+
+Resetting Hasura migrations
+===========================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Introduction
+------------
+
+This page explains how to reset the state of migrations and create new migrations from the state that is on the server. 
+This can be useful if the current migration state on your local machine is corrupted.
+
+Step 1: Delete the local migrations
+-----------------------------------
+
+.. note::
+
+   Before deleting the local migrations, you might consider storing them in a backup folder in case you need them later using this command: ``mv -r migrations migrations_bckup``
+   
+Also, if you do not wish to create a back up folder, you can use the following command to delete all the contents in the ``migrations`` directory.
```

Comment:
This is not either or. We have to delete the contents anyway to clear migrations. Regardless of if we create a backup folder.